src/main/java/com/microsoft/aad/adal4j/AdalAuthorizatonGrant.java

```diff
@@ -62,6 +62,12 @@
         if (this.params != null) {
             outParams.putAll(this.params);
         }
+        
+        if(!outParams.containsKey("scope"))
+        {
+        	outParams.put("scope", "openid");

```

Comment:
ADAL .NET behaves just like this where it simply assigns scope=openid. We also do not allow scopes to be set by the user. I do not see the case/need to deal with multi scope value. what do you think?